Nos/Assets/Localization/Localizable.xcstrings

```diff
@@ -144,7 +144,7 @@
         "ko" : {
           "stringUnit" : {
             "state" : "translated",
-            "value" : "Nos는 Planetary를 개발한 팀이 Nostr 프로토콜을 기반으로 만든 새로운 소셜 미디어 앱입니다. 알고리즘이 아닌 인간을 위해 설계되었습니다. Nos.social에서 자세히 알아보세요."
+            "value" : "Nos는 Planetary를 개발한 팀이 Nostr 프로토콜을 기반으로 만든 새로운 소셜 미디어 앱입니다. 알고리즘이 아닌 인간을 위해 설계되었습니다. Nos.social에서 자세히 알아보세요"
```

Comment:
Nah. This was me trying to figure out how to fix Crowdin. I got it, but this is now outdated and wrong.